gem/templates/malawi/core/related-article-list.html

```diff
@@ -0,0 +1,69 @@
+{% load wagtailcore_tags wagtailimages_tags core_tags %}
+
```

Comment:
Why is this template part of this PR? it doesn't seem to be related to any of this work.

Reply:
@KaitCrawford not sure as to why this was not causing errors before, but the omitting this template will cause 500 errors

Reply:
@moh-moola was it raising errors on your local instance or in the tests?

Reply:
@KaitCrawford locally